x-pack/test/api_integration/apis/content_management/helpers.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { parse as parseCookie } from 'tough-cookie';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export const sampleDashboard = {
+  contentTypeId: 'dashboard',
+  data: {
+    kibanaSavedObjectMeta: {},
+    title: 'Sample dashboard',
+  },
+  options: {
+    references: [],
+    overwrite: true,
+  },
+  version: 2,
+};
+
+export async function loginAsInteractiveUser({
+  getService,
+}: Pick<FtrProviderContext, 'getService'>): Promise<{
+  Cookie: string;
+}> {
+  const config = getService('config');
+  const supertest = getService('supertestWithoutAuth');
+
+  const username = config.get('servers.kibana.username');
+  const password = config.get('servers.kibana.password');
+  const response = await supertest
+    .post('/internal/security/login')
+    .set('kbn-xsrf', 'xxx')
+    .send({
+      providerType: 'basic',
+      providerName: 'basic',
+      currentURL: '/',
+      params: { username, password },
+    })
+    .expect(200);
+  const cookie = parseCookie(response.header['set-cookie'][0])!.cookieString();
+
+  return { Cookie: cookie };
+}
```

Comment:
__note:__ not a blocker for this PR, obviously, but it'd be great if we can start embracing testing with non-admin/non-superuser users when possible 🙂 If you're up for it in this or the next PRs, here are the examples: https://github.com/elastic/kibana/blob/main/x-pack/test/security_api_integration/tests/user_profiles/suggest.ts#L17. I'm also happy to guide if you have any questions or issues with that.

Having said that, glad to see more functional and API integration tests with changes like that. Good job!